google_shopping.js

```diff
@@ -1,5 +1,85 @@
+//1.) The kind of results you're are dealing are shopping#products. 
+//Go through the items and find all results that have kind of shopping#product. 
+//How many are there? Where else is this count information 
+//stored in the search results?
+
 var data = require("./products.json")
+var newArray = [];
+
+for (var i = 0; i < data["items"].length; i++) {
+	if (data["items"][i]["kind"] === "shopping#product") {
+		newArray.push(data["items"][i]["id"]);
```

Comment:
This way is perfectly fine, but you could also make a counter,
```var shoppingProducts = 0;```
and increment it at each match:
```shoppingProducts++```